### PR TITLE
docs/tools: update PD Control and PD Recover documents

### PR DESCRIPTION
**Pull request by lilin90**
Signed-off-by: lilin90 <lilin@pingcap.com>

## What have you changed? (mandatory)

- Updated PD Control (code and description) and PD Recover based on [this PR](https://github.com/pingcap/docs/pull/772)
- Fixed typo

## What are the type of the changes? (mandatory)

- Improvement (non-breaking change which is an improvement to an existing feature)

## Does this PR affect tidb-ansible update? (mandatory)

No.

## Refer to a related PR or issue link (optional)

 [Related PR](https://github.com/pingcap/docs/pull/772)

### CODE REVIEW

File: docs/tools/pd-control.md [hunk 1]
```diff
@@ -406,42 +424,66 @@ Usage:
...
+  "leader": {...},
+  "written_bytes": 251302,
+  "read_bytes": 60472,
+  "approximate_size": 96,
+  "approximate_keys": 524442
 }
 ```
 
-### `region key [--format=raw|pb|proto|protobuf] <key>`
+### `region key [--format=raw|encode] <key>`
```
Comment (rleungx) [inline docs/tools/pd-control.md]: We may need to add `hex` here?

File: docs/tools/pd-control.md [hunk 2]
```diff
@@ -10,10 +10,10 @@ As a command line tool of PD, PD Control obtains the state information of the cl
 
 ## Source code compiling
 
-1. [Go](https://golang.org/) Version 1.9 or later
+1. [Go](https://golang.org/) Version 1.11 or later
 2. In the root directory of the [PD project](https://github.com/pingcap/pd), use the `make` command to compile and generate `bin/pd-ctl`
 
-> **Note:** Generally, you don't need to compile source code as the PD Control tool already exists in the released Binary or Docker. However, dev users can refer to the above instruction for compiling source code.
+> **Note:** Generally, you do not need to compile source code because the PD Control tool already exists in the released Binary or Docker. For developer users, the `make` command can be used to compile source code.
```
Comment (rleungx) [inline docs/tools/pd-control.md]: Is this necessary to change this?

Comment (lilin90) [reply docs/tools/pd-control.md]: @rleungx Yes. The previous version is not clear enough.

File: docs/tools/pd-control.md (hunk 1 above)
Comment (lilin90) [reply docs/tools/pd-control.md]: @disksing What's your opinion about `hex`? @rleungx Would you please tell me exactly what the heading should look like if we do need it? Thanks!

Comment (disksing) [reply docs/tools/pd-control.md]: em, I think we need to update pingcap/docs first to include features about hex.

File: docs/tools/pd-control.md [hunk 3]
```diff
@@ -406,42 +424,66 @@ Usage:
...
 {
-  "region": {
-      "id": 2,
-      ...
-  }
-  "leader": {
-      ...
-  }
+  "id": 2,
+  "start_key": "t\\x80\\x00\\x00\\x00\\x00\\x00\\x00\\xff\\x1d\\x00\\x00\\x00\\x00\\x00\\x00\\x00\\xf8",
```
Comment (siddontang) [inline docs/tools/pd-control.md]: not hex format now?

**Pull request merged**